example/javascriptlib/basic/1-simple/build.mill

```diff
@@ -15,16 +15,7 @@ Hello James Bond Professor
 
 > mill foo.test
 PASS .../foo.test.ts
-...generateUser function
-...should generate a user with all specified fields...
-...should default lastName and role when they are not provided...
-...should default all fields when args is empty...
 ...
-Test Suites:...1 passed, 1 total...
-Tests:...3 passed, 3 total...
```

Comment:
Let's leave the `3 passed, 3 total` line. It's a good assert to make sure that all tests correctly get picked up